### DISCUSSION TITLE
Multi Voice Reading

### DISCUSSION THREAD

**Discussion by larry77** [aedocw/epub2tts#214] [Ideas]
Hello,
I stumbled across this project

https://github.com/DrewThomasson/VoxNovel

which (I quote) 

> leverages the capabilities of booknlp to analyze literature, attribute quotations to specific characters, and generate a tailored audiobook where each character has a distinct voice through coqui tts.

Is this possible already for epub2tts? To be honest, with this feature, it would surpass the overwhelming majority of the audiobooks I listened to, where a single voice covered all the characters.

Thanks for sharing your thoughts.

**Comment by aedocw** [comment]
Wow! I had not seen VoxNovel before, this looks to be way better than epub2tts!

I'll have to play with it. It's unlikely epub2tts will incorporate this feature any time soon, but seems like using VoxNovel would be much better. It's already got a GUI, uses great voice models, and looks like it produces an excellent audiobook.

**Reply by larry77** [reply]
So, are you suggesting we should switch to that ;-) ?
It looks wonderful, but for instance I am on seasoned hardware and with my modest GPU I can get things done in epub2tts, while I am struggling with Vox (which takes nothing away from the project itself; I may need to invest in more up-to-date hardware).

**Comment by aedocw** [comment]
Haha well I'm saying that project looks pretty good and is under active development right now. Vs. epub2tts which I'm still paying attention to but I don't have a lot of spare time to put towards major new features (like implementing a GUI, multiple speaking voices with booknlp, etc)

I do appreciate though that epub2tts works nicely without a GPU. I also like going from text file, which has made it really easy to prep a book before having it read.

I'll continue working on epub2tts but I don't know that I will ever get to a point where I make it quite as fancy as VoxNovel :)

**Comment by k-aito** [comment]
I will add my two cent about it ^^

The other project looks really cool, but the first thing that I read is that booknlp can only work on English text for now.
I will maybe give it a try with a French text, but I think that it will be pretty weird.
